templates/named.conf.erb

```diff
@@ -8,11 +8,6 @@ logging {
                 severity dynamic;
         };
 };
-zone "." IN {
-       type hint;
-       file "<%= @root_hint %>";
```

Comment:
I just confirmed that RedHat does not include the root zone in the rfc1912 file.